docs/examples/screenly/README.md

```diff
@@ -0,0 +1,130 @@
+
+## Installing WoTT agent on Screenly OSE
+
+First you will need a Screenly device if you do not already.
+Follow the instructions to install Screenly OSE (the free version) on your Raspberry Pi [here](https://www.screenly.io/ose/). We reccommend you follow the first option and use something like Etcher to flash the SD card with the Screenly boot. 
+**Note** Screenly will overwrite your OS.
+
+When the Raspberry Pi reboots, it will take you to the Screenly network configuration. You will need to access the SSID as you would a wifi network and enter the password shown. 
+
+![screenly](https://github.com/WoTTsecurity/agent/blob/master/docs/examples/screenly/screenly-setup.jpg)
+
+Then navigate to the [Address](https://www.screenly.io/wifi) shown and login with your network details. This will grant you access to the Screenly management page. 
+
+Now we need to secure access to your Screenly device. If you haven't installed the WoTT Agent already, install [it](https://github.com/WoTTsecurity/agent).
+
+To do this on the Screenly device, you need to access the terminal through `CTRL` + `ALT` + `F1`. To return back to the GUI, it is `CTRL` + `ALT` + `F2`. 
+Once here, follow the WoTT agent installation as you would on any other device. 
+
+**Optional:** If you want to avoid using the terminal directly on your Screenly device in the future, enable SSH through the `sudo raspi-config` command. It is also reccommended that you change your User and Password from the default `pi` and `raspberry` (this will also improve your WoTT security score!) if you are going to do this.
+
+You will now need to register the Pi on the WoTT dashboard. 
+
+
+## Downloading WoTT credentials on Screenly OSE
+
+Login to the WoTT Dash and navigate to 'Claim Device.' If you have installed the WoTT agent recently, the relevant information should be displayed on your terminal screen. If not, use the following commands:
+
+``` 
+$ sudo wott-agent whoami
+$ sudo wott-aget claim-token
+```
+
+This will give you your Device ID and the token value to claim your device. 
+
+Your Raspberry Pi should now be registered. Navigate to your Dashboard and select the Pi:
+
+![pi](https://github.com/WoTTsecurity/agent/blob/master/docs/examples/screenly/rasbpi.png)
+
+On the overview page, add a tag. This will be important for the credentials. This tag can be whatever you want, however we suggest something like `screenly-pi` or `pi` or a variant of that form:
+
+![tag](https://github.com/WoTTsecurity/agent/blob/master/docs/examples/screenly/tag.png)
+
+Now navigate to credentials and add a new credential with the following details:
+
+```
+Name = screenly
+Key = login
+Value = username:password
+Owner = user
```

Comment:
`Owner = pi`

Reply:
I've done owner = user here then specified why in the comments in case the the pi isn't actually named pi. If it's not clear enough, I can change it

Reply:
In Screenly it's always 'pi'. It's hardcoded in many places.